photos.html

```diff
@@ -1,111 +1,203 @@
+  <link rel="stylesheet" href="css/styles.css">
+</head>
+
+<body>
+  <div class="wrapper">
+    <!-- main page -->
+    <header class="page-header">
+      <nav class="main-nav">
+        <ul class="main-nav__list">
+          <li class="main-nav__item main-nav__item--active"><a href="index.html">Home</a></li>
+          <li class="main-nav__item"><a href="photos.html">Photos and videos</a></li>
+          <li class="main-nav__item"><a href="form.html">Feedback Form</a></li>
+          <li class="main-nav__item"><a href="#">HTML Academy</a></li>
+        </ul>
+        <div class="main-nav__toggle main-nav__toggle--close">
+          <div class="main-nav__menu-bar main-nav__menu-bar--top"></div>
+          <div class="main-nav__menu-bar main-nav__menu-bar--middle"></div>
+          <div class="main-nav__menu-bar main-nav__menu-bar--bottom"></div>
+        </div>
+        <a class="main-nav__link" href="index.html"><img class="main-nav__logo" src="images/logo.png"  alt="Sedona logo"></a>
+      </nav>
+
+      <div class="intro intro--smaller header-separator">
+        <h3 class="intro__header">Welcome to gorgeous Sedona. "Because the Grand Canyon sucks!"</h3>
+        <svg class="header-separator__svg" xmlns="http://www.w3.org/2000/svg" version="1.1" width="100%" height="50" viewBox="0 0 100 10" preserveAspectRatio="none">
+              <polygon points="0 2, 50 10, 100 2, 100 500, 0 500"></polygon>
+            </svg>
+      </div>
+    </header>
+
+    <main class="main-content">
+      <section class="preamble">
+        <div class="preamble__inner preamble__inner--high">
+          <h2 class="preamble__header preamble__header--small">Photo and video</h2>
+          <p class="preamble__paragraph preamble__paragraph--high">Can't decide to travel because of the right time? We hope photos will help you forget about politics and economics.</p>
         </div>
-      </header>
+      </section>
 
-      <main class="main-content">
-        <section class="preamble">
-          <div class="preamble__inner">
-            <h2 class="preamble__header">Photo and video</h2>
-            <p class="preamble__paragraph">Can't decide to travel because of the right time? We hope photos will help you forget about politics and economics.</p>
+      <div class="gallery">
+        <section class="gallery__item">
+          <div class="gallery__inner">
+            <div class="gallery__picture">
+              <picture>
+                <source media="(max-width: 640px)" srcset="img/photo-sunset-mobile@1x.jpg 1x, img/photo-sunset-mobile@2x.jpg 2x">
+                <source media="(max-width: 1000px)" srcset="img/photo-sunset-tablet@1x.jpg 1x, img/photo-sunset-tablet@2x.jpg 2x">
+                <source media="(min-width: 1001px)" srcset="img/photo-sunset-desktop@1x.jpg 1x, img/photo-sunset-desktop@2x.jpg 2x">
+                <img src="img/photo-sunset-mobile@1x.jpg" alt="Sunset">
+              </picture>
```

Comment:
Ok, in our case we are building a mobile first responsive layout. I suggest the following:
```
<picture>
 <source media="(min-width: 768px)" srcset="img/photo-sunset-desktop@1x.jpg 1x, img/photo-sunset-desktop@2x.jpg 2x">
 <source media="(min-width: 320px)" srcset="img/photo-sunset-tablet@1x.jpg 1x, img/photo-sunset-tablet@2x.jpg 2x">
 <source srcset="img/photo-sunset-mobile@1x.jpg 1x, img/photo-sunset-mobile@2x.jpg 2x">
 <img src="img/photo-sunset-mobile@1x.jpg" alt="Sunset">
</picture>
```

Reply:
Also you missing the magic:
```
img {
 max-width: 100%; 
 height: auto; 
}
```
If you remember we added the above code in form PR. Pull the changes (rebase with the upstream master)

Reply:
About the <picture> set in your suggestion it will work in this way. 320px and higher will show the tablet picture. For instance, iphone 6 screen has 414 css pixels it means that the browser on this mobile phone will show pictures for the tablet. The same thing happens to the tablet. I will modify a bit the numbers. instead min-width: 320px I will use 480px and the same for the desktop instead of 768 i will use value 1024.